js/content.js

```diff
@@ -0,0 +1,447 @@
+							hideNavBar.innerText = 'Hide Navbar';
+							navBar.style.height = '35px';
+						}
+					});
+					
+					hideNavBar.onclick = function () {
+						if (navBar.hasAttribute('id')) { 
+							navBar.removeAttribute('id','nothidden');
+							navBar.style.height = '0px';
+							hideNavBar.innerText = 'Show NavBar';
+							}
+						else { 
+							navBar.style.height = '35px';
+							navBar.setAttribute('id','nothidden'); 
+							hideNavBar.innerText = 'Hide NavBar';
+							}
+					}
+					
+					addonSettingsPopup('game-view');
+					settingButton.parentNode.appendChild(hideNavBar);
+				})
+				
+				chrome.storage.local.get({'haxMuteConfig' : true}, function (items) {
+					if (items.haxMuteConfig) {
+						muteAll = document.createElement('button');
+						muteAll.style.padding = '5px 10px';
+						muteAll.style.width = '80px';
+						muteAll.innerText = 'Mute';
+						muteAll.onclick = function () { 
+							if (muteAllToggle) {
+								muteAllToggle = false;
+								var chats = gameframe.contentWindow.document.querySelector('[data-hook="log"]').getElementsByTagName('p');
+								for (i = 0; i < chats.length; i++) { chats[i].removeAttribute('hidden'); }
+								muteAll.innerText = 'Mute';
+							}
+							else {
+								muteAllToggle = true;
+								muteAll.innerText = 'Unmute';
+							}
+						}
+					var dividerDiv = document.createElement('div');
+					dividerDiv.style = 'width: 5px';
+					chatInput.parentNode.appendChild(dividerDiv);
+					chatInput.parentNode.insertBefore(muteAll,chatInput);
+					}
+				});
+				
+				chrome.storage.local.get({'haxShortcutConfig' : false}, function (items) {
+					if (items.haxShortcutConfig) {
+						var emojiDoc = document.createElement('button');
+						emojiDoc.style.padding = '5px 10px';
+						emojiDoc.innerText = '😊';
+						emojiDoc.onclick = function () { chrome.runtime.sendMessage({'type': 'emoji'}) };
+						
+						chatInput.parentNode.insertBefore(emojiDoc, chatInput.parentNode.lastChild.previousSibling);
+					}
+				});
+				
+			case tempView == "dialog":
+				chrome.storage.local.get({'haxMuteConfig' : true}, function (items) {
+					if (items.haxMuteConfig) {
+						var popupWait = waitForElement('div.dialog');
+						popupWait.then(function (popup) {
+							var name = popup.firstChild.innerText;
+							if (name === 'Add-on Settings') {
+								return
+							}
+							var muteBtn = document.createElement('button');
+							muteBtn.className = 'mb';
+							popup.insertBefore(muteBtn, popup.lastChild);
+							if (muted.has(name)) {
+								muteBtn.innerText = 'Unmute';
+							}
+							else {
+								muteBtn.innerText = 'Mute';
+							}
+							muteBtn.onclick = function () { 
+								if (muted.has(name)) {
+									muted.delete(name);
+									muteBtn.innerText = 'Mute';
+									}
+								else {
+									muted.add(name);
+									muteBtn.innerText = 'Unmute';
+									}
+							}
+
+							// tag stuff start here
```

Comment:
Need to incorporate latest change from my master branch, replaces nickname spaces with underscores + block for notice